src/main/java/com/github/timm/cucumber/generate/CucumberITGeneratorByScenario.java

```diff
@@ -145,6 +146,7 @@ private void writeContentFromTemplate(final Writer writer) {
         context.put("strict", overriddenParameters.isStrict());
         context.put("featureFile", featureFileLocation);
         context.put("reports", createFormatStrings());
+        context.put("generateByLineNumber", generateByLineNumber);
         if (!config.filterFeaturesByTags()) {
             context.put("tags", overriddenParameters.getTags());
```

Comment:
If we are filtering by scenario, and therefore using line number, then `tags` is never required.  Scenarios have already been filtered by tag at this point.

So I think lines 149/150 are unnecessary.